build.zig.zon

```diff
@@ -4,8 +4,8 @@
     .paths = .{"."},
     .dependencies = .{
         .zine = .{
-            .url = "https://github.com/kristoff-it/zine/archive/03f80646b83cadb2e693ff5d97445d3e16c8e222.tar.gz",
-            .hash = "1220e3e4938edf652776349c45b3bb58774d540a050034488ff8dab7dbe410cc2977",
+            .url = "git+https://github.com/kristoff-it/zine#c4924570fa54ab204bb8b1bf1a140d0cc04ab08a",
```

Comment:
I got this version from this PR: https://github.com/kristoff-it/zine-ssg.io/pull/7